site/data/qasida/sifhu-li-ya-man-ra-ayta.json

```
@@ -0,0 +1,214 @@
+{
+    "name": {
+        "ara": "صفه لي",
+        "en": "Ṣifhu Lī – Describe him to me"
+    },
+    "verses": {
+        "ara": [
+            ["صِفْهُ لِي يَا مَنْ رَأَيْتَ الحِبَّ لَيْلًا"],
+            ["إِنَّ عَيْنِي تَشْتَهِي ذَاكَ الجَمَالَا"],
+
+            ["صِفْ جَمِيلًا كَامِلًا خَلْقًا وَخُلُقًا"],
+            ["مِنْ جَمَالِ الذَّاتِ قَدْ حَازَ الكَمَالَا"],
+
+            ["صِفْ مَلِيحًا طَرْفُهُ أَسْيَى العَوَالِمْ "],
+            ["صِفْ وَجِيهًا وَجْهُهُ حَازَ الكَمَالَا"],
+
+            ["صِفْهُ لِي بِالسُّنْدُسِيَّةِ حِيْنَ يَبْدُو "],
+            ["كَامِلَ الأَوْصَافِ قَدْ مَلَكَ الدَّلَالَا"],
+
+            ["صِفْ عُيُونَ الهَاشِمِي صِفْ لِي المُحَيَّا"],
+            ["صِفْ لِي ثَغْرًا بِابْتِسَامَتِهِ تَلَالًا"],
```

["صِفْ لِي ثَغْراً بِابْتِسَامَتِهِ تَلَالا"],

---

did this one mean to drop the fatha on the end?
ie would it be
```suggestion
            ["صِفْ لِي ثَغْراً بِابْتِسَامَتِهِ تَلَالَا"],
```